developer/contributing/pull-requests.rst

```diff
@@ -23,21 +29,22 @@ When creating a pull request:
  * **review**: when it is reviewable
  * **feedback**: if you have reviewed someone else's pr
 
-We recommend that you can also use the `Waffle board`_ to manage the state of your
-PRs and issues.
+We recommend that you can also use the `Waffle board`_ to manage the state of
+your PRs and issues.
 
 Template
 --------
 
-Template is divided in two parts: *tasks* (for developer) and *information* (for reviewer).
+Template is divided in two parts: *tasks* (for developer) and *information*
+(for reviewer).
 
 The information should include the following fields:
 
-* **Tests pass**: **yes or no**; If tests fail, add reason
```

Comment:
I think its kindof superficial anyway, Symfony does it which is why its here. But really .. if travis is green the tests are passing. This is useless imo.